operator-framework/pom.xml

```diff
@@ -85,9 +85,8 @@
       <scope>test</scope>
     </dependency>
     <dependency>
-      <groupId>io.javaoperatorsdk</groupId>
-      <artifactId>jenvtest</artifactId>
-      <scope>test</scope>
```

Comment:
Shouldn't it still use the test scope?

Reply:
I guess that is inherited from dependency management

Reply:
https://stackoverflow.com/questions/15221299/dependencymanagement-and-scope seems to indicate that putting scopes in `dependencyManagement` is a bad practice…

Reply:
hmm, ok np, will move it